src/ServerScriptService/MarketplaceServiceHandler.server.lua

```diff
@@ -0,0 +1,22 @@
+-- SERVICES
+local MarketplaceService = game:GetService("MarketplaceService")
+local Players = game:GetService("Players")
+local SS = game:GetService("ServerStorage")
+
```

Comment:
Esqueceu de incluir um comentário assim:
```suggestion
-- MEMBERS
```

Reply:
Comentário exemplo: "Não preciso adicionar esse comentário aqui"